odoo/osv/expression.py

```diff
@@ -680,12 +680,14 @@ def push_result(query, params):
                     push(dom_leaf, model, alias)
 
             elif field.type == 'properties':
-                if len(path) != 2 and "." in path[1]:
-                    raise ValueError(f"Wrong path {path}")
+                property_name = None
```

Comment:
I think the real issue is that I use a `and` instead of `or`, the only change should be should be
```py
if len(path) != 2 or "." in path[1]:
    raise ValueError(f"Wrong path {path}")
```

Reply:
Hello @std-odoo ,
I understood your point that it is a typo in the condition. I have made the changes accordingly, please check it.
Thanks :slightly_smiling_face: